src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -69,31 +72,25 @@ dependencies:
 Next, run `pub get packages`, then import the `flutter_localizations` library and specify
 `localizationsDelegates` and `supportedLocales` for `MaterialApp`:
 
-<!-- skip -->
+<?code-excerpt "gen_l10n_example/lib/main.dart (LocalizationDelegatesImport)"?>
 ```dart
 import 'package:flutter_localizations/flutter_localizations.dart';
-// TODO: uncomment the line below after codegen
-// import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+```
 
-// ...
+<?code-excerpt "gen_l10n_example/lib/main.dart (LocalizationDelegates)" replace="/App.*//g"?>
+```dart
+return Material
+  title: 'Localizations Sample
+  localizationsDelegates: [
 
-MaterialApp(
- localizationsDelegates: [
-   // ... app-specific localization delegate[s] here
-   // TODO: uncomment the line below after codegen
-   // AppLocalizations.delegate,
-   GlobalMaterialLocalizations.delegate,
-   GlobalWidgetsLocalizations.delegate,
-   GlobalCupertinoLocalizations.delegate,
- ],
- supportedLocales: [
+    GlobalMaterialLocalizations.delegate,
+    GlobalWidgetsLocalizations.delegate,
+    GlobalCupertinoLocalizations.delegate,
+  ],
+  supportedLocales: [
     const Locale('en', ''), // English, no country code
-    const Locale('ar', ''), // Arabic, no country code
-    const Locale.fromSubtags(languageCode: 'zh'), // Chinese *See Advanced Locales below*
-    // ... other locales the app supports
+    const Locale('es', ''), // Spanish, no country code
   ],
-  // ...
-)
```

Comment:
Should the closing brackets be removed?

Reply:
I changed the script to extract the whole MaterialApp class to avoid confusion.